lib/backbeat/routes.js

```diff
@@ -85,24 +85,32 @@ function routes(redisKeys, allSites) {
             extensions: {},
             method: 'monitoringHandler',
         },
-        // Route: /_/crr/pause/<site-name>
-        // Where <site-name> is an optional field
+        // Route: /_/crr/pause/<site>
```

Comment:
Let's use the word `location` instead of `site` or `site-name`. It's less confusing.